### PR TITLE
Feature/bid by cp id v2

### PR DESCRIPTION
**Pull request by mjoyce91**
Relies on #110 

- Adds a new model of employee-languages
- Adds supplemental employee data to the GET bids by cp_id response
- Adds sorting to the GET bids by cp_id request

### CODE REVIEW

File: src/services/bids.js [hunk 1]
```diff
@@ -26,6 +27,36 @@ async function get_bid(cp_id, perdet_seq_num) {
...
+      'position.cycle',
+      'position.bidstats',
+      'employee',
+      'employee.skills',
+      'employee.languages',
+      'employee.currentassignment'
+    ], require: false })
+
+  let bids$ = bids.map(bid => formatData(bid.serialize()))
+  const orderBy = _.get(query, 'order_by');
```
Comment (scott062) [inline src/services/bids.js:48]: request_params.order_by ?

**Comment by scott062** [comment]
Looks good! It looks like you're providing all the data fields we need, but it's not exact with Pat's anticipated payload. I'm assuming we revisit this once we get the real endpoints? Or change it now?

File: src/services/bids.js (hunk 1 above)
Comment (mjoyce91) [reply src/services/bids.js:48]: Some endpoints actually don't use the `request_params` prefix

File: src/routes.js [hunk 2]
```diff
@@ -64,6 +64,14 @@ var appRouter = function (app) {
     res.end(download)
   });
 
+  app.get("/bids/cycleposition", async function (req, res) {
```
Comment (mjoyce91) [inline src/routes.js:67]: From @scott062 

>cyclePositions/bidders - from Pat's email (still theoretical until we get it though)

**Pull request merged**